projects/dashboards/src/widgets/widget-base.model.ts

```diff
@@ -0,0 +1,18 @@
+import { ModelProperty, STRING_PROPERTY } from '@hypertrace/hyperdash';
+
+export abstract class WidgetBaseModel {
```

Comment:
Could we call this just `BaseModel` (or `Model`)? The idea being this isn't necessarily unique to widget models, but potentially any mode